tests/aequilibrae/project/test_matrices.py

```diff
@@ -29,90 +30,90 @@ def tearDown(self) -> None:
 
-        mat = self.matrices.get_matrix('oMx')
+        mat = self.matrices.get_matrix("oMx")
         mat.computational_view()
-        self.assertEqual(floor(mat.matrix_view.sum()), 20309, 'Matrix loaded incorrectly')
+        self.assertEqual(floor(mat.matrix_view.sum()), 20309, "Matrix loaded incorrectly")
 
     def test_get_record(self):
-        rec = self.matrices.get_record('omx')
-        self.assertEqual(rec.cores, 4, 'record populated wrong. Number of cores')
-        self.assertEqual(rec.description, None, 'record populated wrong. Description')
+        rec = self.matrices.get_record("omx")
+        self.assertEqual(rec.cores, 4, "record populated wrong. Number of cores")
+        self.assertEqual(rec.description, None, "record populated wrong. Description")
 
     def test_record_update_cores(self):
-        rec = self.matrices.get_record('omx')
+        rec = self.matrices.get_record("omx")
         rec.update_cores()
-        self.assertEqual(rec.cores, 2, 'Cores update did not work')
+        self.assertEqual(rec.cores, 2, "Cores update did not work")
 
     def test_save_record(self):
-        rec = self.matrices.get_record('omx')
+        rec = self.matrices.get_record("omx")
 
         text = self.randomword(randint(30, 100))
         rec.description = text
         rec.save()
 
         self.curr.execute('select description from matrices where name="omx";')
-        self.assertEqual(text, self.curr.fetchone()[0], 'Saving matrix record description failed')
+        self.assertEqual(text, self.curr.fetchone()[0], "Saving matrix record description failed")
 
     def test_delete(self):
-        self.matrices.delete_record('omx')
+        self.matrices.delete_record("omx")
         self.curr.execute('select count(*) from matrices where name="omx";')
 
-        self.assertEqual(0, self.curr.fetchone()[0], ' Deleting matrix record failed')
+        self.assertEqual(0, self.curr.fetchone()[0], " Deleting matrix record failed")
 
         with self.assertRaises(Exception):
-            self.matrices.get_record('omx')
+            self.matrices.get_record("omx")
 
     def test_list(self):
         df = self.matrices.list()
-        self.curr.execute('select count(*) from Matrices')
-        self.assertEqual(df.shape[0], self.curr.fetchone()[0], 'Returned the wrong number of matrices in the database')
+        self.curr.execute("select count(*) from Matrices")
+        self.assertEqual(df.shape[0], self.curr.fetchone()[0], "Returned the wrong number of matrices in the database")
 
-        self.assertEqual(df[df.status == 'file missing'].shape[0], 1, 'Wrong # of records for missing matrix files')
+        self.assertEqual(df[df.status == "file missing"].shape[0], 1, "Wrong # of records for missing matrix files")
 
     def __mat_count(self, should_have: int, error_message: str) -> None:
-        self.curr.execute('Select count(*) from Matrices;')
+        self.curr.execute("Select count(*) from Matrices;")
```

Comment:
You should do one PR where you just run black over the entire codebase and then never worry about " vs ' again

Reply:
Will do